pkg/deploy-manager/storagecluster.go

```diff
@@ -47,7 +47,8 @@ func (t *DeployManager) StartDefaultStorageCluster() error {
 	return nil
 }
 
-func defaultStorageCluster() (*ocsv1.StorageCluster, error) {
+// GetDefaultStorageCluster returns a default StorageCluster manifest
+func GetDefaultStorageCluster() (*ocsv1.StorageCluster, error) {
```

Comment:
using "Get" in this function name might be confusing. Someone might easily assume it's doing a client GET on the apiserver for the default storage cluster.

maybe we could just drop the "Get" and export it as "DefaultStorageCluster"?

Reply:
`DefaultStorageCluster` is already in use and so I had to append `Get`. Any other suggestions?